client/src/App.js

```diff
@@ -1,40 +1,37 @@
-import "./App.css";
-import "./styles/layouts.css";
+import './App.css';
+import './styles/layouts.css';
 
-import React from "react";
-import Header from "./components/Header";
-import Footer from "./components/Footer";
-import Home from "./pages/Home";
-import About from "./pages/About";
-import EventsPage from "./pages/EventsPage";
-import EventDetailContainer from "./components/Events/EventDetailContainer";
-import Signup from "./pages/Signup";
-import Signin from "./pages/Signin";
-import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import React from 'react';
+import Header from './components/Header';
+import Footer from './components/Footer';
+import Home from './pages/Home';
+import About from './pages/About';
+import EventsPage from './pages/EventsPage';
+import EventDetailContainer from './components/Events/EventDetailContainer';
+import Signup from './pages/Signup';
+import Signin from './pages/Signin';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
```

Comment:
Usually, linting changes should be a separate PR to make it easier to review actual changes. For now, don't worry about it.

Reply:
is there a prettier script that changes all double quotes to single? Wait, we have a prettierrc.yaml file, and it says `singleQuote: true`.